prime-dotnet-webapi/HttpClients/Verifiable Credential/VerifiableCredentialClient.cs

```diff
@@ -251,6 +251,32 @@ public async Task<JObject> GetPresentationProof(string presentationExchangeId)
             return body;
         }
 
+        public async Task<bool> DeleteCredentialAsync(Credential credential)
+        {
+            _logger.LogInformation("Deleting credential cred_ex-Id={id}", credential.CredentialExchangeId);
+
+            HttpResponseMessage response = null;
+            try
+            {
+                response = await _client.DeleteAsync($"issue-credential/records/{credential.CredentialExchangeId}");
+            }
+            catch (Exception ex)
+            {
+                await LogError(response, ex);
+                throw new VerifiableCredentialApiException("Error occurred attempting to delete a credential: ", ex);
+            }
+
+            if (!response.IsSuccessStatusCode)
+            {
+                await LogError(response);
+                throw new VerifiableCredentialApiException($"Error code {response.StatusCode} was provided when calling VerifiableCredentialClient::DeleteCredentialAsync");
```

Comment:
I feel that this should probably just return false rather than throw. I see a very small possibility that a user has one credential that cant be deleted for some reason and returns an error 4XX status code; which would prevent us from deleting/revoking the rest of their credentials when we call RevokeCredentialsAsync